doc/whats_new/v1.0.rst

```diff
@@ -262,6 +262,13 @@ Changelog
 :mod:`sklearn.ensemble`
 .......................
 
+- |Enhancement| :class:`~sklearn.ensemble.HistGradientBoostingClassifier` and
+  :class:`~sklearn.ensemble.HistGradientBoostingRegressor` take cgroups quotas
+  into account when deciding the number of threads used by OpenMP. This
+  avoids performance problems caused by over-subscription when using those
+  classes in a docker container for instance. :pr:`20477`
```

Comment:
> when using those classes in a docker container for instance

This is unrelated to docker, isn't it?
Some docker images will set CPU quotas, but some don't. I think that the over-generalization to all docker containers is confusing.

Reply:
Also, this only affects linux machines right?

Reply:
> Also, this only affects linux machines right?

This will affect most machines because: the docker deamon needs a Linux kernel (to use cgroups and other features of it); this kernel which will the host's OS's, or a virtual machine's running Linux.

I think Docker started developing support for Windows-native images, but this is rather niche.

Reply:
My comment about linux was dissociated from the one about docker.

Basically, we should probably clarify that this change does not affect Windows or OSX users.

Reply:
I think that it still affect Windows or OSX users because a Linux VM has to be used for Docker in those cases.

Reply:
I am not talking about docker.

I am saying that as far as I understand, this entry will not affect users using scikit-learn on Windows or OSX.
(as long as they don't use docker, which is probably the vast majority of them).